Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -1355,6 +1355,7 @@ void TBWidget::InvokePointerUp(int x, int y, MODIFIER_KEYS modifierkeys, bool to
 {
     int ox = x;
     int oy = y;
+    TBWidget *old_capture;
```

Comment:
Ok, let's document a little of what is going on though, for ox, oy, and old_capture variables, thanks!